nucypher/characters/lawful.py

```diff
@@ -116,54 +117,97 @@ def generate_kfrags(self, bob: 'Bob', label: bytes, m: int, n: int) -> List:
         delegating_power = self._crypto_power.power_ups(DelegatingPower)
         return delegating_power.generate_kfrags(bob_pubkey_enc, self.stamp, label, m, n)
 
-    def create_policy(self, bob: "Bob", label: bytes, m: int, n: int, federated=False):
+    def create_policy(self,
+                      bob: "Bob",
+                      label: bytes,
+                      m: int,
+                      n: int,
+                      federated: bool = False,
```

Comment:
I'm very skeptical here that this is a perhaps toxic moisture creeping in.  If the `Alice` is federated only, then this being `True` is impossible representation, isn't it?

Reply:
Hmm, perhaps this parameter can be removed altogether somehow...